### PR TITLE
README.md - remove dollar signs

### PR DESCRIPTION
**Pull request by JanneKiiskila**
# Mbed Edge pull request

## Description

Remove dollar signs from the cli examples, as they are completely
unnecessary and just make the copy paste things tedious (line by line
vs. whole section). The formatting already makes it obvious those
were meant for the command line.

## Test instructions

N/A, cosmetic `README.md` change only.

### API change(s)

 - [x] Not applicable.
 - [ ] API is backwards compatible.
 - [ ] API documentation is updated.

### Example applications updated

 - [x] Not applicable.

### CODE REVIEW

**Comment by JanneKiiskila** [comment]
Please review @OPpuolitaival  @ygoyal18 .

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ sudo apt install doxygen graphviz
 
 Fetch the Git submodules that are direct dependencies for Edge.
 ```
```
Comment (OPpuolitaival) [inline README.md:65]: Add tip what is the language?
```suggestion
```bash
```

**Review by OPpuolitaival — APPROVED** [review]

**Comment by JanneKiiskila** [comment]
Did it now through the whole doc @OPpuolitaival .

**Review by OPpuolitaival — APPROVED** [review]

**Comment by JanneKiiskila** [comment]
Someone else needs to merge @OPpuolitaival  @ygoyal18 .

**Review by ygoyal18 — APPROVED** [review]
LGTM

**Comment by JanneKiiskila** [comment]
Rebased, force-pushed.

**Pull request merged**